server/blob/blob-export-file/src/main/java/org/apache/james/blob/export/file/LocalFileBlobExportMechanism.java

```diff
@@ -86,7 +87,7 @@ private String copyBlobToFile(BlobId blobId) {
 
 
             String fileName = RandomStringUtils.random(STRING_LENGTH, WITH_LETTERS, !WITH_NUMBERS);
-            String fileURL = configuration.exportDirectory + "/" + fileName;
+            String fileURL = configuration.exportDirectory + "/" + fileName + ARCHIVE_FILE_EXTENSION;
```

Comment:
/!\ The sharing mechanism shares a blob, whatever it is.

You are guessing that it is a zip but this `blob` could be anything.

We should modify the BlobExportMechanism to allow the caller specifying the extension, no?